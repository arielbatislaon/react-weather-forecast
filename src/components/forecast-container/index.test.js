import React from 'react';
import renderer from 'react-test-renderer';

import ForeCastContainer from './index';



it('renders ForeCastContainer correctly ', () => {
  const forecasts = [{
    applicable_date: '2021-09-19',
          min_temp: '12.5',
          max_temp: '32.5',
  }];
  const city ='London';
  const tree = renderer.create(<ForeCastContainer forecasts={forecasts} city={city} />).toJSON();
  expect(tree).toMatchSnapshot();
});