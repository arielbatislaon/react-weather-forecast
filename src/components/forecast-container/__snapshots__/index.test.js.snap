// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ForeCastContainer correctly  1`] = `
<div>
  <h4>
    City of London Min/Max Temperature Forecast
  </h4>
  <div
    className="row"
  >
    <div
      className="forecast-item col"
    >
      <div>
        <h5>
          Sunday
        </h5>
      </div>
      <div>
        Min Temp.
        :
        12.5
      </div>
      <div>
         
        Max Temp.
        :
        32.5
      </div>
    </div>
  </div>
</div>
`;
